raiden/tests/unit/transfer/test_node.py

```diff
@@ -54,3 +72,81 @@ def test_is_transaction_effect_satisfied(chain_state, token_network_id, netting_
     iteration = state_transition(chain_state=chain_state, state_change=channel_settled)
 
     assert is_transaction_effect_satisfied(iteration.new_state, transaction, state_change)
+
+
+def test_get_network(chain_state, token_network_id):
+    token_address = factories.make_address()
+    token_network = TokenNetworkState(address=token_network_id, token_address=token_address)
+    payment_network = PaymentNetworkState(
+        address=factories.make_address(), token_network_list=[token_network]
+    )
+    chain_state.identifiers_to_paymentnetworks[payment_network.address] = payment_network
+    assert get_networks(
+        chain_state=chain_state,
+        payment_network_identifier=payment_network.address,
+        token_address=token_address,
+    ) == (payment_network, token_network)
+
+
+def test_subdispatch_invalid_initiatortask(chain_state, token_network_id):
+    subtask = object()
+    chain_state.payment_mapping.secrethashes_to_task[UNIT_SECRETHASH] = subtask
+    transition_result = subdispatch_initiatortask(
+        chain_state=chain_state,
+        state_change=None,
+        token_network_identifier=token_network_id,
+        secrethash=UNIT_SECRETHASH,
+    )
+    assert isinstance(transition_result, TransitionResult)
```

Comment:
Do we want to keep such checks, `mypy` should check that.

Reply:
The return type can be determined by mypy, right.